lib/src/strings.dart

```diff
@@ -207,10 +207,18 @@ const windowsDylibLocations = {
   r'C:\Program Files\LLVM\bin\',
 };
 const macOsDylibLocations = {
+  // Default Xcode commandline tools installation.
+  '/Library/Developer/CommandLineTools/usr/',
+  // Default path for LLVM installed with apt-get.
   '/usr/local/opt/llvm/lib/',
+  // Default path for LLVM installed with brew.
   '/opt/homebrew/opt/llvm/lib/',
-  '/Library/Developer/CommandLineTools/usr/',
+  // Default Xcode installation.
+  // Last because it does not include ObjectiveC headers by default.
```

Comment:
I don't think "last" matters since https://github.com/dart-lang/ffigen/pull/278 changed macOsDylibLocations to use a Set instead of List, right?

(I was just checking if that change had made it into stable yet and noticed this new comment.)

Reply:
The default set implementation maintains order (LinkedHashSet) - https://api.dart.dev/stable/2.17.3/dart-core/Set-class.html

Reply:
Neat! TIL. Thanks!